scripts/opx_build

```diff
@@ -17,7 +17,15 @@
 
 set -e
 
-DIST=jessie
+DIST=$(lsb_release --codename --short)
```

Comment:
Eventually I think we'll have to be able to specify both DIST and ARCH from the command line (and have docker files that can support building for more than just one target).  This means the container's OS image isn't necessarily going to be the same as the target.  But until then, I'm OK with this change.